### PR TITLE
Add support for nested properties (fixes #27)

### LINKED ISSUE #27: Add support for nested properties
Currently, it doesn't seem to be possible to specify the lat/lon fields if they are nested, e.g.:

```
{
    "id": 1,
    "location": {
        "latitude":"51.20837",
        "longitude":"0.22978"
    },
    "category":"some-data
}
```

An interface for this would be something like:

`GeoJSON.parse(data, {Point: ['location.latitude', 'location.longitude']});`

I wonder whether one of the solutions outlined here would be workable:
http://stackoverflow.com/questions/2631001/javascript-test-for-existence-of-nested-object-key

### PR DESCRIPTION
**Pull request by mvl22**

### CODE REVIEW

**Comment by mvl22** [comment]
I release this code licensed as public domain.

File: geojson.js [hunk 1]
```diff
@@ -157,6 +157,24 @@
         geom.type = gtype;
         geom.coordinates = [Number(item[val[1]]), Number(item[val[0]])];
       }
+      
+      // Geometry parameter specified as: {Point: ['container.lat', 'container.lng']}
+      else if(Array.isArray(val) && /^.+\..+$/.test(val[0]) && /.+\..+$/.test(val[1])){
```
Comment (nmccready) [inline geojson.js]: I would make this a function in the file describing what the regex is doing.

```
function isNested(val) {
  return  /^.+\..+$/.test(val);
}
```

**Comment by nmccready** [comment]
I went further with this to support some google stuff easier.

https://github.com/caseypt/GeoJSON.js/compare/master...realtymaps:nested?expand=1

**Comment by mvl22** [comment]
Great work - I look forward to seeing this merged!

**Comment by caseypt** [comment]
Thank you both for the PRs. It may take me a while to review #29 as it's pretty big, but I think it makes sense to support nested properties.

File: geojson.js (hunk 1 above)
Comment (danactive) [inline geojson.js]: Agreed

**Comment by caseycesari** [comment]
Superceded by #29. Thanks for the PR!

**Pull request closed without merging**